pkg/remote/resource_list_error_handler.go

```diff
@@ -0,0 +1,35 @@
+package remote
+
+import (
+	"fmt"
+
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/cloudskiff/driftctl/pkg/alerter"
+	"github.com/sirupsen/logrus"
+)
+
+func HandleResourceListError(err error, alertr *alerter.Alerter) error {
```

Comment:
`HandleResourceEnumerationError`

Reply:
Should we also change the name of the file ?

Reply:
Indeed